src/custom_nodes/east_ocr/config.json

```diff
@@ -3,20 +3,23 @@
         {
             "config": {
                 "name": "east",
-                "base_path": "/models/east_icdar2015_resnet_v1_50_rbox"
+                "base_path": "/OCR/east_fp32",
+                "plugin_config": {
+                    "CPU_THROUGHPUT_STREAMS":  "1"
```

Comment:
Why setting streams to 1?

Reply:
This significantly increases pipeline performance, east inference time is reduced from 900ms to 150ms.

Reply:
Both latency & throughput?
I am fine with change just wasn't sure why it is needed.